src/scm/gitlab.js

```diff
@@ -0,0 +1,278 @@
+"use strict";
+
+class Gitlab {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/5becb274ac17be98fee6e945e23cbab998e9d302/src/scm/gitlab.js#L3"></span>‘Gitlab’ is defined but never used.</td>
    </tr>
  </tbody>
</table>